nwbinspector/checks/image_series.py

```diff
@@ -38,3 +38,15 @@ def check_image_series_external_file_relative(image_series: ImageSeries):
                     "Please adjust the absolute path to be relative to the location of the NWBFile."
                 )
             )
+
+
+@register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=ImageSeries)
+def check_image_series_too_large(image_series: ImageSeries, gb_lower_bound: float = 20.0):
+    """Check if an ImageSeries stored is larger than gb_lower_bound and suggests external file."""
+    data = image_series.data
+    data_size_gb = data.size * data.dtype.itemsize // 1e9
+    if data_size_gb > gb_lower_bound:
+        return InspectorMessage(
+            severity=Severity.HIGH,
```

Comment:
I'd only use `Severity` if you have two checks like this but where one is slightly more impactful than the other, such as the data compression checks: https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/nwbinspector/checks/nwb_containers.py#L11-L58

Reply:
So, severity should be removed of the check? OK, done.